jiwonjang/app.js

```diff
@@ -0,0 +1,40 @@
+const dotenv = require("dotenv");
+dotenv.config();
```

Comment:
```suggestion
require("dotenv").config();
```
위와 같이 한줄로 작성될 수도 있습니다! 기존 코드와 한 번 비교해보세요 :)